scripts/release/index.js

```diff
@@ -38,6 +38,7 @@ cooker.cook('publish', [
       cook.pushTagToRemote,
       cook.createReleaseNotes(releaseNotesTemplate),
       cook.createGithubRelease,
+      cook.runNpmScript('deploy'),
```

Comment:
Have you tested it? I am not sure this will work. At least website dependencies is not installed automatically